Sources/Conduit/Auth/TokenStorage/TokenMigrator.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  TokenMigrator.swift
+//  Conduit
+//
+//  Created by Eneko Alonso on 2/20/19.
+//  Copyright © 2019 MINDBODY. All rights reserved.
```

Comment:
This is over a year ago. Copy and paste? As this is open source I wonder if this needs to be correct

Reply:
Yes, this is copied/moved from CheckIn app